config/packages/notifier.yaml

```diff
@@ -1,6 +1,14 @@
 framework:
     notifier:
         chatter_transports:
+            zulip: '%env(ZULIP_DSN)%'
```

Comment:
Do you need this if transports are dynamic?

Reply:
These settings won't create any problems I think. If there is ever sending from an internal DSN I prefer to keep it. These environment variables are not defined.